test/e2e/kubevirt/net.go

```diff
@@ -62,3 +88,11 @@ func GenerateGatewayMAC(node *corev1.Node, networkName string) (string, error) {
 
 	return util.IPAddrToHWAddr(lrpJoinIP).String(), nil
 }
+
+func GenerateGatwayIPv6RouterLLA(node *corev1.Node, networkName string) (string, error) {
+	lrpAddresses, err := util.ParseNodeGatewayRouterJoinAddrs(node, networkName)
+	if err != nil {
+		return "", err
+	}
+	return util.HWAddrToIPv6LLA(util.IPAddrToHWAddr(lrpAddresses[0].IP)).String(), nil
```

Comment:
IMHO you need to assert the nodes have join addresses. Either in the caller, or here, within the helper.

And please remind me: the join IP addresses will **always** be the first entry in the list, right ?

Reply:
> IMHO you need to assert the nodes have join addresses. Either in the caller, or here, within the helper.

ok. 


And please remind me: the join IP addresses will **always** be the first entry in the list, right ?

This is the node annotation for join addreses it just has join addreses this is not the LRP network addreses.